pytorch_blade/src/ltc/disc_compiler/replay.cpp

```diff
@@ -0,0 +1,152 @@
+// Copyright 2022 The BladeDISC Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ltc/disc_compiler/replay.h"
+
+#include <sys/stat.h>
+#include <torch/csrc/jit/api/function_impl.h>
+#include <torch/csrc/jit/passes/constant_propagation.h>
+#include <torch/csrc/lazy/ts_backend/ts_backend_impl.h>
+#include <torch/script.h>
+#include <fstream>
+
+#include "compiler/jit/tool_funcs.h"
+#include "ltc/disc_compiler/disc_compiler.h"
+
+namespace torch_disc {
+namespace compiler {
+
+void FoldOutputs(std::shared_ptr<torch::jit::Graph> graph) {
+  auto return_node = graph->return_node();
+  auto tuple = return_node->input()->node();
+  return_node->removeAllInputs();
```

Comment:
A little bit confused ... why input nodes of `return` get removed and then added?

Reply:
A simple description will be added in this PR:

before:

``` text
%4 = prim::TupleConstruct(%1, %2, %3)
return %4
``` 

after:

```text
return %1, %2, %3
```